rhel7/profiles/anssi_nt28_enhanced.profile

```diff
@@ -1,9 +1,15 @@
 documentation_complete: true
 
-title: 'DRAFT - ANSSI DAT-BP28 (enhanced)'
+title: 'ANSSI-BP-028 (enhanced)'
 
-description: 'Draft profile for ANSSI compliance at the enhanced level. ANSSI stands for Agence nationale de la sécurité des
-    systèmes d''information. Based on https://www.ssi.gouv.fr/.'
+description: |-
+    This profile contains configurations that align to ANSSI-BP-028 at the enhanced hardening level.
+
+    ANSSI is the French National Information Security Agency, and stands for Agence nationale de la sécurité des systèmes d''information.
```

Comment:
sorry, just noticed. You have double apohstrophe in the full name of ANSSI

Reply:
Double apostrophe are also in description of other levels

Reply:
This is to escape the single quote.

Reply:
@yuumasato In generated guide, it shows as double. Copy-paste guide:
> ANSSI is the French National Information Security Agency, and stands for Agence nationale de la sécurité des systèmes d''information.

Reply:
Right, the behavior changed as the descriptions is not enclosed in single quotes anymore.

Reply:
Updated in c4b11df